models/marts/core/int_all_graph_resources.sql

```diff
@@ -61,10 +61,10 @@ joined as (
         end as model_type_prefix,
         case 
             when unioned_with_calc.resource_type in ('test', 'source', 'metric', 'exposure', 'seed') then null
-            when {{ dbt.position('naming_convention_folders.folder_name_value','unioned_with_calc.directory_path') }} = 0 then null
+            when {{ dbt.position("concat('/',naming_convention_folders.folder_name_value,'/')",'unioned_with_calc.directory_path') }} = 0 then null
```

Comment:
We also need to make it work for Windows users who have their files delimited with `\`.
There is [already a macro](https://github.com/dbt-labs/dbt-project-evaluator/blob/8a4c7e484a9ba81733c293d72ffeb447536a772a/macros/get_directory_pattern.sql#L2-L10) that you should be able to use that provides the file delimiter based on the OS being used.